app/views/components/p.scala.html

```diff
@@ -14,10 +14,16 @@
  * limitations under the License.
  *@
 
-@import utils.Utils.emptyString
+@import utils.Utils.{emptyString, period}
+@import play.twirl.api.HtmlFormat
 
 @this()
 
 @(content: Html, id: Option[String] = None, classes: Option[String] = None, link: Option[Html] = None, tabLink: Option[Html] = None)(implicit messages: Messages)
 
-<p @{id.fold(emptyString)(id => s"id=$id")}  class="@{classes.fold("govuk-body")(classes => s"$classes")}">@content @link @tabLink</p>
+@l = @{ link.fold(Html(""))(l => HtmlFormat.fill(Seq(Html(" "), l))) }
```

Comment:
(on lines 24-27) Better to replace "" with emptyString and better to create a val (like singleSpace in Utils) for " " and make use of it